docs/examples/tunnel-binding-simple/README.md

```diff
@@ -0,0 +1,94 @@
+# Using Cloudflared as the reverse proxy
+
+> This builds on [the getting started guide](../../install.md), and it is recommended to read that first.
+
+This example shows how to configure cloudflared to route to two (or more) applications you are hosting.
+
+You can of course configure only one application, but two are shown in this example so that it can be generalised.
+
+```
++-----------------------+
+|    internet users     |
++-----------------------+
+            |
+            v
++-----------------------+
+|   cloudflare tunnel   |
++-----------------------+
+     |             |
+     v             v
+ +-------+     +-------+ 
+ | app 1 |     | app 2 | 
+ +-------+     +-------+ 
+```
+
+## Motivation
+
+Cloudflared is designed to be able to work as a reverse proxy in its own right, routing directly to your applications.
+
+This is the simplest configuration one can follow and does not need an additional reverse proxy to handle routing.
+All routing is handled by TunnelBinding resources managed by this controller.
+
+## Steps
+
+> All steps assume you are working from the directory this README is in.
+
+> **Ensure that you have kubectl installed.**
+
+To create a ClusterTunnel, we need to store Cloudflare credentials in a Secret. Follow the steps below. Have a look at the [detailed configuration guide](configuration.md) for more details.
```

Comment:
```suggestion
To create a ClusterTunnel, we need to store Cloudflare credentials in a Secret. Follow the steps below. Have a look at the [detailed configuration guide](../../configuration.md) for more details.
```

Did you mean to refer `../../configuration.md`?

Reply:
good catch, thanks!